timesketch/lib/aggregators/interface.py

```diff
@@ -236,8 +237,8 @@ def elastic_aggregation(self, aggregation_spec):
         Returns:
             Elasticsearch aggregation result.
         """
-        # pylint: disable=unexpected-keyword-arg
-        aggregation = self.elastic.search(
+        # pylint: disable=unexpected-keyword-arg, no-value-for-parameter
+        aggregation = self.elastic.aggregation_shim(
```

Comment:
Do we need a separate method for this? It is only used in one call from this file.
I suggest you just do:

```
aggregation = self.elastic.client.search( ... )
```

Reply:
and remove `aggregation_shim`

Reply:
Thanks for reviewing! The intention of this shim was to move the method from `timesketch.lib.aggregators.interface` into the `timesketch.lib.datastores.elastic.ElasticDatastore` class. 

In line 157 of `timesketch.lib.aggregators.interface` we currently use a direct call to the Elastic Library, rather than using ElasticDatastore. I wanted to standardise this, rather than reimplementing the check for SSL config here, so re-wrote it as ElasticDatastore. Unfortunately (I may be wrong here) the implementation of ElasticDataStore.Search doesn't match the implementation of Elasticsearch.search - hence the shim method. 

I was thinking of moving the method `timesketch.lib.aggregators.interface.elastic_aggregation` into the `timesketch.lib.datastores.elastic.ElasticDatastore` class, but didn't want to break anything (hence leaving this there currently).

Reply:
My apologies - realised my mistake! moving this to the single call